src/components/MapSelect/style.scss

```diff
@@ -30,6 +30,7 @@
   .zoom-control {
     background-color: #FEC813;
     color: $ams-zwart;
+    font-family: Avenir Next LT W01, arial, sans-serif;
```

Comment:
Wouldn't it make more sense to apply this style rule to the `StyledMap` component in `index.js`?

Reply:
Good point thanks!